fmt.sh

```diff
@@ -110,12 +110,13 @@ run dade-copyright-headers "$dade_copyright_arg" .
 # We do test hlint via Bazel rules but we run it separately
 # to get linting failures early.
 if [ "$hlint_diff" = "true" ]; then
-    changed_haskell_files="$(git diff --name-only origin/master | grep '\.hs$' || [[ $? == 1 ]])"
-    if [ "" != "$changed_haskell_files" ]; then
-        hlint -j4 $changed_haskell_files
-    fi
+  #  We do not run on deleted files, or files that have been added since we last rebased onto trunk.
+  changed_haskell_files="$(git diff --name-only --diff-filter=d "$(git merge-base origin/master HEAD)" | grep '\.hs$' || [[ $? == 1 ]])"
```

Comment:
Any opinion on `--diff-filter=AM`?

Reply:
I'm fine with that. It has the added benefit of not running on untracked files, which shouldn't be a problem.

Reply:
We probably want `--diff-filter=ACMRT`.

Reply:
:+1: